cmd/adsysd/integration_tests/testdata/PolicyUpdate/states/old-data/apparmor.d/adsys/users/adsystestuser@example.com

```diff
@@ -0,0 +1,6 @@
+^adsystestuser@example.com {
```

Comment:
It seems to me that the assets in assets.db are the same for apparmor between the old assets and the new ones. I would have expected:
- some assets to appear and disappear between old and new
- one common asset with changed content.

Based on length, this is not the case if I’m correct? (Sorry, I have to attach it here because you can’t comment on binary diff)

Reply:
privileged_user which is applied between old and new is still the same or am I seeing that wrong? (ack on the other files being different though)